### PR TITLE
Update Docker Desktop release notes for 2.3.6.1

### PR DESCRIPTION
**Pull request by djs55**
## Proposed changes

We're about to release an edge version of Desktop: 2.3.6.1

### CODE REVIEW

**Review by usha-mandya — DISMISSED** [review]
LGTM. Thanks.

**Comment by usha-mandya** [comment]
@djs55 Do we need to add Compose 1.27.0 to the release notes?

**Review by usha-mandya — DISMISSED** [review]
LGTM

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+
+> [Download](https://desktop.docker.com/mac/edge/47792/Docker.dmg)
+
+### Upgrades
+
+- [Docker Compose 1.27.0](https://github.com/docker/compose/releases/tag/1.27.0)
+
+### Bug fixes and minor changes
+
+-  Docker Desktop now correctly displays the default state of "Use gRPC FUSE for file sharing" in the UI.
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:25]: I suggest delete the word "default".

Comment (stephen-turner) [reply docker-for-mac/edge-release-notes.md:25]: Also there is a bug number for this ... https://github.com/docker/for-mac/issues/4864

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**